### PR TITLE
Corrected Restyled Procedures - Part 2

### PR DESCRIPTION
**Pull request by zbig01**
made needed changes to types.php, rtl changes and themes updated
accordingly

below added by bradymiller:
Up for grabs demo for this PR is here: https://www.open-emr.org/wiki/index.php/Development_Demo#Beta_-_Up_For_Grabs_Demo

### CODE REVIEW

**Comment by bradymiller** [comment]
Up for grabs demo for this PR is here:
https://www.open-emr.org/wiki/index.php/Development_Demo#Beta_-_Up_For_Grabs_Demo

File: Documentation/help_files/configure_orders_help.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
         @media only screen and (max-width: 768px) {
            [class*="col-"] {
            width: 100%;
-           text-align:left!important;
+           /*text-align:left!important;*/
```
Comment (bradymiller) [inline Documentation/help_files/configure_orders_help.php]: better to remove than comment out

File: Documentation/help_files/procedure_provider_help.php [hunk 2]
```diff
@@ -24,7 +24,7 @@
         @media only screen and (max-width: 768px) {
            [class*="col-"] {
            width: 100%;
-           text-align:left!important;
+           /*text-align:left!important;*/
```
Comment (bradymiller) [inline Documentation/help_files/procedure_provider_help.php]: better to remove than comment out

File: interface/orders/types_edit.php [hunk 3]
```diff
@@ -252,388 +253,467 @@ function proc_type_changed() {
...
-                            <div class="col-sm-12 label-div">
-                                <label class="control-label" for="form_description"><?php echo xlt('Description'); ?>:</label><a href="#description_info" class="icon-tooltip" data-toggle="collapse"><i class="fa fa-question-circle" aria-hidden="true"></i></a>
+                            <div class="row">
+                                <div class="col-sm-12 ordonly  fgponly foronly">
+                                    <div class="clearfix">
+                                        <div class="col-sm-12 label-div">
+                                            <label class="control-label" for="form_lab_id"><?php echo xlt('Order From'); ?>:</label><a href="#order_info" class="icon-tooltip" data-toggle="collapse"><i class="fa fa-question-circle" aria-hidden="true"></i></a>
+                                        </div>
+                                        <div class="col-sm-12">
+                                            <?php
```
Comment (bradymiller) [inline interface/orders/types_edit.php]: PLACEMARKER for ongoing code review

File: Documentation/help_files/configure_orders_help.php (hunk 1 above)
Comment (zbig01) [reply Documentation/help_files/configure_orders_help.php]: I will,  just left it till I was sure it did not affect the regular LTR views in responsive  mode

File: interface/orders/types_edit.php [hunk 4]
```diff
@@ -252,388 +253,467 @@ function proc_type_changed() {
...
+                                            } else {
+                                                $title = xl('The entity performing this procedure');
+                                                $ord_disabled = '';
+                                            }
+                                            ?>
+                                            <select name='form_lab_id' id='form_lab_id' class='form-control <?php echo $ord_disabled ;?>'
+                                                title='<?php echo attr($title); ?>'>
+                                                <?php
+                                                if ($ordtype) {
+                                                    $ppres = sqlStatement("SELECT ppid, name FROM procedure_providers WHERE ppid = $labid " . "ORDER BY name, ppid");
```
Comment (bradymiller) [inline interface/orders/types_edit.php]: need to use sql binding on this:
```
$ppres = sqlStatement("SELECT ppid, name FROM procedure_providers WHERE ppid = ? ORDER BY name, ppid", array($labid));
```

**Comment by zbig01** [comment]
Made requested changes. 

After you bring this in I can rtl the other help files and pages with the help icon :)

**Comment by bradymiller** [comment]
hi @zbig01 ,
I reread this code and don't see any reverted code issue (like the prior PR), so gonna bring this into the codebase. Thanks again for the awesome work!
-brady

**Pull request merged**